regions/core/core.py

```diff
@@ -161,6 +161,36 @@ def to_shapely(self):
         """
         raise NotImplementedError("")
 
+    @abc.abstractmethod
+    def make_patch(self, **kwargs):
+        """Convert to mpl patch
+
+        Returns
+        -------
+        patch : `~matplotlib.patches.Patch`
+            Matplotlib patch
+        """
+        raise NotImplementedError
+
+    def plot(self, ax=None, **kwargs):
+        """
+        Calls make_patch method forwarding all kwargs and adds patch
+        to given axis.
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes`, optional
+            Axis
+        """
+        import matplotlib.pyplot as plt
+
+        ax = plt.gca() if ax is None else ax
```

Comment:
Could this be usefully shortened to `ax = ax or plt.gca()`?  It may be less readable than the ternary, though, so just thumbs-down this if you don't like it.